docs/architecture/domain.md

```diff
@@ -0,0 +1,117 @@
+## Definition Model
+
+![Definition](assets/definition-model.png)
```

Comment:
Yeah but from the graph it doesn't look clear what "workflow" is. Maybe have a rectangle around all those 4 jobs and caption it as "workflow"?